pymemcache/pool.py

```diff
@@ -72,10 +74,18 @@ def get(self):
                 obj = self._obj_creator()
                 self._used_objs.append(obj)
                 return obj
-            else:
-                obj = self._free_objs.pop()
-                self._used_objs.append(obj)
-                return obj
+            obj = None
+            while self._free_objs:
+                # FIFO try to remove timeout idle objects
```

Comment:
Let's move this comment up outside of the loop and describe the full behavior of this change: it both locates a free object and expires idle ones.